server/routerlicious/packages/services-ordering-rdkafka/src/resourcesFactory.ts

```diff
@@ -71,6 +71,7 @@ export class RdkafkaResourcesFactory implements IResourcesFactory<RdkafkaResourc
 		const automaticConsume = config.get("kafka:lib:rdkafkaAutomaticConsume");
 		const consumeTimeout = config.get("kafka:lib:rdkafkaConsumeTimeout");
 		const maxConsumerCommitRetries = config.get("kafka:lib:rdkafkaMaxConsumerCommitRetries");
+		const consumeLoopTimeoutDelay = config.get("kafka:lib:rdkafkaConsumeLoopTimeoutDelay");
```

Comment:
@andrewstanovsky - Please review. I think you added something similar.

Reply:
approved